Neos.Eel/Classes/Utility.php

```diff
@@ -17,6 +17,18 @@
  */
 class Utility
 {
+    /**
+     * Return the expression if it is an valid EEL expression, otherwise return null.
+     *
+     * @param string $expression
+     *
+     * @return bool
```

Comment:
incorrect return type, should be `?string`

Reply:
I changed it to `string|null` to keep compatibility with PHP 7.0